index.js

```diff
@@ -117,8 +122,7 @@ QUnit.notifications = function( options ) {
         }
       }, false );
 
-      toolbar.appendChild( notification );
-      toolbar.appendChild( label );
+      checkboxContainer.appendChild( label );
```

Comment:
this line depends on #30 
It assumes that checkbox is already wrapped with a label